wordpress-mobile/WordPress-iOS
# Better offline messages in Comments list

In the spirit of small PRs, this one implements part of #5759 : comments list.

To test:
1. Enter flight mode
2. Navigate to any site > Comments
3. If there is content cached from a previous visit, it should be visible, but there should be an alert warning that connection is offline:
![img_2605](https://user-images.githubusercontent.com/2722505/36953452-b931b7ac-1fcf-11e8-96b6-247d5f97fda7.PNG)

4. If there is no content cached, there should be also an alert, and when the alert is dismissed, there should be an error message in the no results view, warning that the connection is offline.
5. In this case, pulling to refresh will display the same alert and the same error message.
![img_2606](https://user-images.githubusercontent.com/2722505/36953466-d723568a-1fcf-11e8-8761-0c44e847c3c4.PNG)
![img_2607](https://user-images.githubusercontent.com/2722505/36953469-dcff174c-1fcf-11e8-80b8-5727e2b2d0b4.PNG)


6. After the connection is back up, pulling to refresh should load data again.

Open question. Should I try to make the content reload automatically after the connection is back up? I would like to give it a shot, but I am not sure if it is out of scope.
